### PR TITLE
Add the libedit package explicitly 

### LINKED ISSUE #36: COPY from CSV does not work
When trying to import data from a CSV file I'm able to upload my CSV to a /tmp location inside the container, but running the import itself with this file selected produces the following errors in the status:

```
Error loading shared library libedit.so.0: No such file or directory (needed by /usr/local/bin/psql)
Error relocating /usr/local/bin/psql: stifle_history: symbol not found
Error relocating /usr/local/bin/psql: rl_completion_matches: symbol not found
Error relocating /usr/local/bin/psql: add_history: symbol not found
Error relocating /usr/local/bin/psql: write_history: symbol not found
Error relocating /usr/local/bin/psql: read_history: symbol not found
Error relocating /usr/local/bin/psql: rl_filename_completion_function: symbol not found
Error relocating /usr/local/bin/psql: current_history: symbol not found
Error relocating /usr/local/bin/psql: readline: symbol not found
Error relocating /usr/local/bin/psql: previous_history: symbol not found
Error relocating /usr/local/bin/psql: using_history: symbol not found
Error relocating /usr/local/bin/psql: rl_initialize: symbol not found
Error relocating /usr/local/bin/psql: history_set_pos: symbol not found
Error relocating /usr/local/bin/psql: next_history: symbol not found
Error relocating /usr/local/bin/psql: rl_attempted_completion_function: symbol not found
Error relocating /usr/local/bin/psql: rl_line_buffer: symbol not found
Error relocating /usr/local/bin/psql: rl_readline_name: symbol not found
Error relocating /usr/local/bin/psql: rl_basic_word_break_characters: symbol not found
```

### PR DESCRIPTION
**Pull request by alsin**
So that psql does not fail when running a COPY .. FROM on a CSV file

Fixes #36

### CODE REVIEW

**Comment by alsin** [comment]
I've concatenated two RUN statements into single one to reduce the number of layers. Tested on my machine, importing CSVs now works.

**Comment by chrisob** [comment]
@thaJeztah Any thoughts on this PR?

**Comment by DoryZi** [comment]
Any clue what's going on with this??

File: Dockerfile [hunk 1]
```diff
@@ -4,11 +4,10 @@ ENV PGADMIN_VERSION=3.0 \
     PYTHONDONTWRITEBYTECODE=1
 
 # Install postgresql tools for backup/restore
-RUN apk add --no-cache postgresql \
+RUN apk add --no-cache libedit postgresql \
  && cp /usr/bin/psql /usr/bin/pg_dump /usr/bin/pg_dumpall /usr/bin/pg_restore /usr/local/bin/ \
- && apk del postgresql
-
-RUN apk add --no-cache alpine-sdk postgresql-dev \
```
Comment (thaJeztah) [inline Dockerfile:11]: I don't think we should combine these steps; having these layers separate improves caching (when building the image), and allows image layers to be shared (if you have multiple versions of this image).

Given that there's no overlap between the two steps, combining the layers won't make the image smaller;

```
REPOSITORY   TAG        IMAGE ID            CREATED             SIZE
pgadmin4     libedit    6a52eb68dd5e        7 seconds ago       243MB
pgadmin4     latest     f188f718d788        3 minutes ago       243MB
```

**Review by thaJeztah — CHANGES_REQUESTED** [review]
thanks @alsin ! I left a comment below; currently working on tagging some new versions of this image, so I'll carry your change to move fast, and to add the `libedit` package 🤗

**Pull request closed without merging**